build/Settings.cs

```diff
@@ -181,7 +187,13 @@ public class SignInfo
                 "Microsoft.AI.*.dll",
                 "Microsoft.Build.Framework.dll",
                 "Microsoft.Build.dll",
-                "Microsoft.CodeAnalysis.*.dll",
+                "Microsoft.CodeAnalysis.CSharp.Scripting.dll",
+                "Microsoft.CodeAnalysis.CSharp.Workspaces.dll",
+                "Microsoft.CodeAnalysis.Razor.dll",
+                "Microsoft.CodeAnalysis.Scripting.dll",
+                "Microsoft.CodeAnalysis.VisualBasic.dll",
+                "Microsoft.CodeAnalysis.VisualBasic.Workspaces.dll",
+                "Microsoft.CodeAnalysis.Workspaces.dll",
```

Comment:
Why'd you have to expand these out? I see wildcards elsewhere in the list still...

Reply:
Wild card was covering a lot more files than actually needed. So expanded them. After doing further testing i found that more than 400 files being signed where as we only needed to sign 220 files. A lot of files were being re-signed. I added code to dynamically remove any files that do not need to be signed. But left this one as is.